fem/bilinearform.hpp

```diff
@@ -888,7 +920,7 @@ class MixedBilinearForm : public Matrix
    virtual const Operator *GetOutputRestriction() const
    { return test_fes->GetRestrictionMatrix(); }
 
-   /** For partially conforming trial and/or test FE spaces, complete the
+   /** @brief For partially conforming trial and/or test FE spaces, complete the
        assembly process by performing A := P2^t A P1 where A is the internal
```

Comment:
```suggestion
       assembly process by performing \f$ A := P2^t A P1 \f$ where A is the internal
```
I think the ambiguitiy of A might be confusing for new users here (as above)